k-distribution/src/main/scripts/bin/kore-print

```diff
@@ -139,6 +140,12 @@ do
       trap - INT TERM EXIT
       ;;
 
+      -v| --verbose)
+      verboseFlag=" --verbose"
+      verbose=1
+      shift
+      ;;
```

Comment:
```suggestion
      -v| --verbose)
      verboseFlag="--verbose"
      verbose=1
      shift
      ;;
```

Is the extra space here on purpose? Maybe it's needed and I don't understand.

Reply:
Just a typo. Thanks!